README.md

```diff
@@ -1 +1,285 @@
+-   `failed-tests` - Will annotate those sections of your code that failed test
+
+## Outputs
+
+By default, action attaches comment to a pull request or commit. However, if you want to use other action for publishing report, you can specify `output: report-markdown`:
+
+```yaml
+- uses: step-security/jest-coverage-report-action@v2
+    # give the id for the step, to access outputs in another step.
+    id: coverage
+    with:
+        # tell to the action to not attach comment.
+        output: report-markdown
+- uses: marocchino/sticky-pull-request-comment@v2
+    with:
+        # pass output from the previous step by id.
+        message: ${{ steps.coverage.outputs.report }}
+```
+
+Also, you can use this data on other platforms. For instance, you can send report to your [Slack](https://github.com/slackapi/slack-github-action) or [Jira](https://github.com/atlassian/gajira-comment).
+
+> **Note**: Working examples of integrations with different platforms are much appreciated! Feel free to open a [PR](https://github.com/step-security/jest-coverage-report-action/pulls).
+
+Available options are:
+* `comment` - Attach comment to PR or commit, depending on event type, which triggered an action.
+* `report-markdown` - Generate output "report", with report contents in markdown format.
+
+Also, you can combine these options:
+
+```yml
+with:
+    # This will attach comment to a PR and generate markdown output.
+    output: comment, report-markdown
+```
+
+## Pull Request Number
+
+If you are using the `push` event to trigger this action, by default it does not know which PR to comment on or the base branch of the PR to compare code coverage with.
+
+You can pass the `prnumber` to the action so that coverage change can be run and comments will be updated on each push, instead of creating a new comment with each run of the action.
+
+You can find the PR number with a number of methods, the [jwalton/gh-find-current-pr](https://github.com/jwalton/gh-find-current-pr) action makes it easy:
+
+```yml
+name: 'coverage'
+on:
+    push:
+        branches:
+            - master
+            - main
+jobs:
+    coverage:
+        permissions:
+            checks: write
+            pull-requests: write
+            contents: write
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v1
+            - uses: jwalton/gh-find-current-pr@v1
+                id: findPr
+            - uses: step-security/jest-coverage-report-action@v2
+                with:
+                   prnumber: ${{ steps.findPr.outputs.number }}
+```
+
+## Customizing report title
+
+If you're running this action multiple times (for instance, when dealing with monorepos), you'll need to distinguish reports from different runs. To do so, you can use the `custom-title` property:
+
+```yaml
+with:
+    custom-title: Coverage report for backend
+```
+
+## License
```

Comment:
can you please remove license part from readme, since there is already a license file?

Reply:
done